tools/nodejs/src/duckdb_node.cpp

```diff
@@ -12,15 +12,26 @@ NodeDuckDB::NodeDuckDB(Napi::Env env, Napi::Object exports) {
 	statement_constructor = node_duckdb::Statement::Init(env, exports);
 	query_result_constructor = node_duckdb::QueryResult::Init(env, exports);
 
-	exports.DefineProperties({
-	    DEFINE_CONSTANT_INTEGER(exports, node_duckdb::Database::DUCKDB_NODEJS_ERROR, ERROR) DEFINE_CONSTANT_INTEGER(
+	auto token_type_enum = Napi::Object::New(env);
+	token_type_enum.Set("IDENTIFIER", 0);
```

Comment:
We should probably also expose the inverted mapping as well, to be a "true" typescript enum

Reply:
Hm, yes, I'll look into that.

Reply:
Done.